f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -117,12 +118,34 @@ def aggregate_stack_data(stack, manifest_file, ecosystem, deps):
         "packages": license_score_list
     }
 
+    _logger = logging.getLogger(__name__)
```

Comment:
Please use instantiated logger in `BaseTask` (available under `self.log`). We configure logging at one place so we are consistent.